packages/react-native-sdk/src/components/ActiveCall.tsx

```diff
@@ -40,10 +40,6 @@ export const ActiveCall = (props: ActiveCallProps) => {
   useEffect(() => {
     // when the component mounts, we ask for necessary permissions.
     verifyAndroidBluetoothPermissions();
-    return () => {
-      // ensure that if this component is unmounted, the call is left.
-      activeCallRef.current?.leave();
-    };
   }, []);
```

Comment:
why removed though?

Reply:
If you do android hardware back or ios default back swipe. does it leave the call? can you check that please

If not, this code will handle that.. may be we can have a condition to check if it is already left and not to do in that case

Reply:
no that will be buggy as the useEffect dep in calling state

better to get calling state from `activeCallRef.current?.state.callingState`